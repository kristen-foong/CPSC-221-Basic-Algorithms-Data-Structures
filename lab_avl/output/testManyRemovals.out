~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1mTesting Many Removals[22m
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rotateRight
rotateRight
rotateRight
rotateRight
rotateRight
rotateLeft
rotateRight
rotateRight
rotateRight
rotateLeft
rotateRight
                                                ______________ 94 _____________                                                  
                                 ______________/                               \______________                                   
                        ______ 87 _____                                                        96                                
                 ______/               \______                                                                                   
            __ 42 _                            92                                                                                
         __/       \__                                                                                                           
       17              78                                                                                                        
     /   \           /   \                                                                                                       
   3       41      45      82                                                                                                    
          /                                                                                                                      
         36                                                                                                                      


