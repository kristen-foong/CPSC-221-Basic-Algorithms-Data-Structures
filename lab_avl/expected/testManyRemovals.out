~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1mTesting Many Removals[22m
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rotateRight
rotateRight
rotateRight
rotateRight
rotateRightLeft
rotateRight
rotateLeft
rotateLeft
rotateLeftRight
rotateLeft
rotateRight
rotateRight
rotateLeft
                        ______ 45 _____                          
                 ______/               \______                   
            __ 26 _                         __ 87 _              
         __/       \__                   __/       \__           
       17              36              78              93        
     /   \           /   \           /   \           /   \       
   3       23      31      42      74      82      92      96    
  / \     / \             /                                      
 1   11  18  24          41                                      


